docs/logicapps-ai-course/02_build_conversational_agents/05-add-user-context-to-tools.md

```diff
@@ -48,56 +48,42 @@ Many solutions use mixed authorization methods. For example, a solution might re
 
 In conversational agent workflows, support for OBO authorization applies only to *shared* managed connectors that use delegated, per-user connections to work with Microsoft services or systems.
 
+> [!WARNING]
+> For debugging purposes, user chat history is set up to be visible by the administrator in the monitoring view. We are looking for feedback to improve this experience in the future.
+> 
+> Outside of monitoring view, each user chat history is private. One user cannot read or continue another user's conversation in the chat client or through A2A protocol.
+
 ## Prerequisites
 
 - An Azure account and subscription. If you don't have a subscription, [sign up for a free Azure account](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
 
 - A Standard logic app resource and conversational agent workflow from previous modules.
 
-  - To use OBO authorization, the logic app resource requires that you set up Easy Auth, previously known as App Service Authentication, on your Standard logic app resource.
-
-    For more information, see the following articles:
-
-    - [Authentication and authorization in Azure App Service and Azure Functions](https://learn.microsoft.com/azure/app-service/overview-authentication-authorization)
-    - [Configure your App Service or Azure Functions app to use Microsoft Entra sign-in](https://learn.microsoft.com/azure/app-service/configure-authentication-provider-aad)
-
-  - The conversational agent workflow requires a connector operation that works with a Microsoft service or system. The connection must also support delegated, per-user connections, for example, Microsoft 365, SharePoint, or Microsoft Graph. If required, the connection might also need tenant administrator consent.
-
-    > [!TIP]
-    > 
-    > Make sure to keep tool descriptions concise and include guidance about the data that they access,
-    > for example, such a description might say "Gets the signed-in user's next five calendar events."
-
-- The chat client interface integrated with conversational agent workflows.
+  - The conversational agent workflow requires a connector operation that works with a Microsoft service or system. The connection must also support delegated, per-user connections, for example, Microsoft 365, SharePoint, or Microsoft Graph.
 
-  This chat client handles token delegation for per-user connections. If you want to use a custom client, the client must pass the user token to the agent.
+- The chat client interface integrated with conversational agent workflows. For this module, we will use the integrated chat client provided by the logic app. You will learn how to use a custom client in [Module 09](./09-deploy-agents-clients.md).
```

Comment:
Put any additional information into a new paragraph. This keeps the list items easy to scan.
- Avoid using "we" as much as possible. It might sound friendly, but keep the focus on the customer or use a neutral voice.
- Actually, I don't know why the client interface is listed as a prerequisite. The customer doesn't have to take any action to set up or install this item. I'd just remove it to reduce noise and not confuse people.

```suggestion
```